src/spectrecoff/Canvas.fs

```diff
@@ -0,0 +1,59 @@
+[<AutoOpen>]
+module SpectreCoff.Canvas
+
+open Spectre.Console
+
+type Width = Width of int
+[<RequireQualifiedAccess>]
+module Width =
+    let unwrap (Width w) = w
+
+type Height = Height of int
+[<RequireQualifiedAccess>]
+module Height =
+    let unwrap (Height h) = h
+
+type ColumnIndex = ColumnIndex of int
+type RowIndex = RowIndex of int
+type StartIndex = StartIndex of int
+type EndIndex = EndIndex of int
+
+type PixelPosition = int*int
+type Pixel = PixelPosition*Color
+type Pixels = 
+    | SinglePixel of PixelPosition
+    | MultiplePixels of PixelPosition list
+    | Row of RowIndex
+    | RowSegment of RowIndex*StartIndex*EndIndex
+    | Column of ColumnIndex
+    | ColumnSegment of ColumnIndex*StartIndex*EndIndex
+    | Rectangle of int*int*int*int
+
+let canvas width height =
+    Canvas (Width.unwrap width, Height.unwrap height) 
+    
+let private setPixel (canvas: Canvas) (pixel: Pixel) = 
+    canvas.SetPixel(fst (fst pixel), snd (fst pixel), snd pixel) |> ignore
+    ()
```

Comment:
I don't think these are needed, since `ignore` returns `unit` already? I am happy to be enlightened in my ignorance in case I am wrong though

Reply:
True, done, thx.